docs/feature_rgblight.md

```diff
@@ -41,6 +41,8 @@ Changing the **Hue** cycles around the circle.<br>
 Changing the **Saturation** moves between the inner and outer sections of the wheel, affecting the intensity of the color.<br>
 Changing the **Value** sets the overall brightness.<br>
 
+![QMK Color Wheel with HSV Values](https://i.imgur.com/vkYVo66.jpg)
```

Comment:
```suggestion
Note that while usually the HSV color space uses 0 – 360° for hue, in QMK the scale is from 0 – 255. You can convert by calculating `255 * Hue / 360`. Here is a color wheel in the 0 – 255 hue range for reference:
![QMK Color Wheel with HSV Values](https://i.imgur.com/vkYVo66.jpg)
```

Reply:
The new picture illustrates only about Hue. So, I think it's worth keeping the previous picture.

Reply:
why not have both.  1 has the values you can assign to actually use.  The other just explains how the colors are defined but doesn't have any number to associate to them.

Reply:
Would it be possible to have this as an svg or at least a png with transparent background?

Reply:
Given the usefulness of this in general, should we get this in in its current form, and fix it up later?

Reply:
Given the usefulness of this in general, should we get this in in its current form, and fix it up later?

Reply:
I think so, yeah.